main.go

```diff
@@ -560,7 +566,7 @@ func createDatabase(project, instance, dbName string, conv *internal.Conv, out *
 	// Spanner DDL doesn't accept them), and protects table and col names
 	// using backticks (to avoid any issues with Spanner reserved words).
 	// We also exclude foreign keys from the schema sent to Spanner.
-	schema := conv.GetDDL(ddl.Config{Comments: false, ProtectIds: true, ForeignKeys: true})
+	schema := conv.GetDDL(ddl.Config{Comments: false, ProtectIds: true, ForeignKeys: false})
```

Comment:
I think this is worth a comment. How about:

// The database is created without foreign key constraints. We will add them later once
// data has been written to the database to avoid foreign key data dependencies during
// data writes.

Reply:
updated comment as mentioned